### ISSUE TITLE
inheritance makes content move to the left side

### ISSUE THREAD

**Issue by eisenherzz** [getgrav/grav-theme-quark#15]
Quark theme works as expected as long I do not use inheritance. When try to build upon quark the content of each page is not centered any more.

**Comment by mettix** [comment]
+1

**Comment by eisenherzz** [comment]
`grid-size: grid-lg`
added to myquark.yaml as a workaround

**Comment by eisenherzz** [comment]
https://discourse.getgrav.org/t/theme-inheritance-default-yaml-settings-inheritance/3515

**Comment by rhukster** [comment]
Yes, this is a limitation with inheritance, as it still expects an overridden theme to have it's own configuration yaml file.

**Comment by mettix** [comment]
@rhukster weird that only this theme has an issue with it, I have tried it with several others, but haven't noticed the same behavior. (I could be wrong offcourse)

**Comment by eisenherzz** [comment]
as far as I understand there are no style informations at the other
themes you would notice immediately.

As I am not a developer I do not know if it is possible just to copy the
original YAML and attach the new stuff via grav-plugin-devtools.

At least this behaviour should be documented. Quark is the new default
and so more people will run in this issue.

Am 13.04.2018 um 15:31 schrieb mettix:

**Comment by tomceek** [comment]
Same problem here, I cannot get by defaut {{ grid_size }} to print anything to myquark theme. as I am not a developer it is quit a pain.

**Comment by eisenherzz** [comment]
Additional, logo has to be deleted that my grav-logo.svg in the inherited theme works

**Comment by tribut** [comment]
The svg inheritance is fixed in https://github.com/getgrav/grav-theme-quark/pull/30

**Comment by eisenherzz** [comment]
Thx for the notification.
